rke/k8s_rke_system_images.go

```diff
@@ -8374,6 +8374,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out of band post v2.7.1
```

Comment:
Nit: this needs to be `post v2.7.2`

Reply:
Addressed in https://github.com/rancher/kontainer-driver-metadata/pull/1110/commits/5b25798542eab0b4cdb306179109f8d58d8ac3dc